### PR TITLE
Notifications into master (for Delhi Release).

### PR DESCRIPTION
**Pull request by akramtexas**
`Merging notifications into Master branch of Delhi`

### CODE REVIEW

**Comment by akramtexas** [comment]
`Blackbox integration tests passing 100% for notifications.`

File: internal/support/notifications/init.go [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+	"fmt"
+	"strings"
+
+	"github.com/edgexfoundry/edgex-go/internal/pkg/consul"
+	"github.com/edgexfoundry/edgex-go/pkg/clients/logging"
+	"github.com/edgexfoundry/edgex-go/internal/support/notifications/interfaces"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/config"
+	"github.com/edgexfoundry/edgex-go/internal"
+	"sync"
```
Comment (tsconn23) [inline internal/support/notifications/init.go]: Organize these imports while you're in there.

File: cmd/support-notifications/main.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/edgexfoundry/edgex-go"
+	"github.com/edgexfoundry/edgex-go/internal"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/usage"
+	"github.com/edgexfoundry/edgex-go/pkg/clients/logging"
```
Comment (tsconn23) [inline cmd/support-notifications/main.go]: Move to bottom

File: cmd/support-notifications/main.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+	"github.com/edgexfoundry/edgex-go/internal/support/notifications"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/startup"
+)
+
+var loggingClient logger.LoggingClient
```
[the remark this answers is not in the document]
Comment (akramtexas) [reply cmd/support-notifications/main.go]: Done (Made all tweaks mentioned in the comments and pushed the updates to remote branch).

**Pull request merged**